airflow/api_fastapi/execution_api/app.py

```diff
@@ -58,6 +58,7 @@ def custom_openapi() -> dict:
             description=app.description,
             version=app.version,
             routes=app.routes,
+            servers=app.servers,
```

Comment:
This was needed to get correct URL i.e. have `/execution` path in the URL.

<img width="648" alt="image" src="https://github.com/user-attachments/assets/fc140bcc-1cad-45d5-9606-ce1dc0962ffe">

Reply:
<img width="488" alt="image" src="https://github.com/user-attachments/assets/d055be97-cbd3-4cf6-9de7-136a0dcba39d">

Reply:
Without it:

<img width="509" alt="image" src="https://github.com/user-attachments/assets/4f6733e8-81bf-45c0-885f-2fa66cb469fe">